docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -25,52 +28,42 @@ configuration file located directly in the `res` directory should be
 considered the default configuration profile. Sub-directories will
 contain configurations appropriate to the respective profile.
 
-With the exception of the `config-seed` service, which will be discussed
-in a moment, a service's configuration profile can be indicated using
-one of the following command line flags:
+As of the Geneva release, it is recommended to utilize environment variable
+overrides rather than creating profiles to override some subset of config
+values. You can see examples of this in the related docker-compose [files](https://github.com/edgexfoundry/developer-scripts/tree/master/releases/nightly-build/compose-files).
```

Comment:
Link need to point to Geneva released compose files.

Reply:
@lenny-intel Geneva compose files don't exist yet. Are you saying this PR can't be merged until they are created?

Reply:
Agreed we'll update the link in separate PR once Geneva compose files are available